src/modules/wrapper/wrapper.component.ts

```diff
@@ -6,6 +6,7 @@ import { StachePageAnchorComponent } from '../page-anchor/page-anchor.component'
 import { StacheNavLink } from '../nav/nav-link';
 
 @Component({
+  // tslint:disable-next-line:component-selector-name
```

Comment:
I'd prefer to place all linting configuration at the top of the file, to not clutter up the presentation.
```
/* tslint:disable:component-selector-name */
...
```